lib/utils.pm

```diff
@@ -954,7 +954,13 @@ sub ensure_serialdev_permissions {
     # ownership has effect immediately, group change is for effect after
     # reboot an alternative https://superuser.com/a/609141/327890 would need
     # handling of optional sudo password prompt within the exec
-    assert_script_run "chown $testapi::username /dev/$testapi::serialdev && gpasswd -a $testapi::username \$(stat -c %G /dev/$testapi::serialdev)";
+    if (sle_version_at_least('12')) {
+        assert_script_run "chown $testapi::username /dev/$testapi::serialdev && gpasswd -a $testapi::username \$(stat -c %G /dev/$testapi::serialdev)";
+    }
+    else {
+        # Backwards support for SLES11
+        assert_script_run "chown $username /dev/$serialdev";
```

Comment:
I guess there is no gpasswd or stat command on SLE 11, right? If yes, make the comment more detailed why it is version specific.

Reply:
yes, add comment with details should be helpful.